intertwine/utils/jsonable.py

```diff
@@ -306,7 +306,7 @@ def form_path(cls, base, *fields):
 
     @classmethod
     def jsonify_value(cls, value, kwarg_map=None, _json=None):
-        '''
+        """
```

Comment:
Change a linter config or something?

Reply:
I wasn't aware of PEP257 back when I standardized on ''' for docstrings. That PEP says, "For consistency, always use """triple double quotes""" around docstrings."
https://www.python.org/dev/peps/pep-0257/

I'll do a global find/replace one of these days...